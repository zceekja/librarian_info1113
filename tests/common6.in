add collection book1.csv
add member Tony Stark
add member Peter parker
add member Steven G. Rogers
rent 100000 111111
rent 100001 111112
relinquish 100000 111111
relinquish 100001 111112
rent 100002 111111
relinquish 100002 111111
common 100000 100001 100002