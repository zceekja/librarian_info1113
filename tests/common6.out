user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: No common books.

user: 